packages/fast-colors/src/color-converters.ts

```diff
@@ -42,6 +47,19 @@ export function contrastRatio(a: ColorRGBA64, b: ColorRGBA64): number {
     return (luminanceB + 0.05) / (luminanceA + 0.05);
 }
 
+/**
+ * Get the luminance of a color in the linear RGB space.
+ * This is not the same as the relative luminance in the sRGB space for WCAG contrast calculations. Use rgbToLuminance instead.
+ * @param rgb The input color
+ */
+export function rgbToLuminanceLinear(rgb: ColorRGBA64): number {

```

Comment:
If we make the above change, this would probably go to `rgbToLinearLuminance` - could do both of them with the modifier after "Luminance" too